grug/app/src/app.rs

```diff
@@ -1,5 +1,7 @@
+use crate::AppCtx;
 #[cfg(feature = "abci")]
 use grug_types::{JsonDeExt, JsonSerExt};
+
```

Comment:
Make sure the format the `use` statements correctly... sometimes the formatter isn't smart enough to do it